nntrainer/nntrainer_logger.cpp

```diff
@@ -77,7 +78,12 @@ Logger::Logger() {
      << std::setw(2) << now->tm_sec << ".out";
   outputstream.open(ss.str(), std::ios_base::app);
   if (!outputstream.good()) {
-    throw std::runtime_error("Unable to initialize the Logger!");
+    char buf[256];
+    char *ret = getcwd(buf, 256);
```

Comment:
https://en.cppreference.com/w/cpp/filesystem/current_path might be better for C++ code.

Reply:
okay, thank you.

Reply:
I've already tried that 12days ago.
We don't use filesystem::current_path becase of android c++ version.
nntrainer already use getcwd().
so I used getcwd() instead of std::filesystem::current_path()
error :
/var/www/html/android/android-ndk-r21e/sources/cxx-stl/llvm-libc++/include/filesystem:1458: undefined reference to `std::__ndk1::__fs::filesystem::__current_path(std::__ndk1::error_code*)'